dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/HibernateRelationshipStore.java

```diff
@@ -88,6 +90,49 @@ public HibernateRelationshipStore(
     super(entityManager, jdbcTemplate, publisher, Relationship.class, aclService, true);
   }
 
+  @Override
+  public Optional<TrackedEntity> findTrackedEntity(UID trackedEntity) {
+    @Language("hql")
+    String hql =
+        """
+                from TrackedEntity te
+                where te.uid = :trackedEntity
+                """;
+    List<TrackedEntity> trackedEntities =
+        getQuery(hql, TrackedEntity.class)
+            .setParameter(TRACKED_ENTITY, trackedEntity.getValue())
```

Comment:
```suggestion
            .setParameter("trackedEntity", trackedEntity.getValue())
```

it's easier to read IMHO as the query itself has the parameter literal in the query string

Reply:
I completely agree.
Reversing it.
I automatically fixed the code smells here but it doesn't make sense in this case.